docs/book/src/cronjob-tutorial/testdata/project/dist/chart/values.yaml

```diff
@@ -1,10 +1,29 @@
 # [MANAGER]: Manager Deployment Configurations
 controllerManager:
   replicas: 1
+  pod:
+    # imagePullSecrets for pulling images from private registries
+    imagePullSecrets: []
```

Comment:
I think I liked it 👍 less the pod above since all here would be only about controller-manager
See : https://github.com/kubernetes-sigs/kubebuilder/issues/4912
The person made a valid feedback about how we are exposing the values

I think we will need to change that we might need a PR just to change what we have today on that
but I want to share it with you.